src/util/Util.cpp

```diff
@@ -160,3 +161,10 @@ void Util::writeCoordinateString(OutputStream* out, double xVal, double yVal) {
     g_ascii_formatd(coordString.data(), G_ASCII_DTOSTR_BUF_SIZE, Util::PRECISION_FORMAT_STRING, yVal);
     out->write(coordString.data());
 }
+
+void Util::systemWithMessage(const char* command) {
+    if (std::system(command) != 0) {
+        string msg = FS(_F("Error executing system command: {1}") % command);
```

Comment:
Can you add the errorcode
```
if (auto errc = std::system(command); errc != 0){
string msg = FS(_F("Error executing system command: {1} with Error {2}") % command % errc);
}
```

Reply:
Hallo Fabian, I did your suggested changes, with a slightly different message.  I also squashed the commits.

Reply:
Ok thanks, this looks good for me, but you created a bit of a mess with your squash attempt.
I fixed it for you.